applications/serial_lte_modem/doc/CARRIER_AT_commands.rst

```diff
@@ -31,27 +31,27 @@ These may be the following:
 
 * ``#XCARRIEREVT: 1,<status>``
 
-  Request to set the modem to full functional mode.
+  Request to set the modem to full functional mode, i.e. ``AT+CFUN=1``.
   ``<status>`` returns two possible values:
 
-  * ``0`` - Request handled successfully.
-  * ``-1`` - Request handling was deferred and must be performed by the application at its earliest convenience.
+  * ``0`` - request handling is fulfilled automatically.
```

Comment:
Uppercase R is missing on every line that you modified.

Reply:
This is not resolved.

Reply:
Ah, just locally, next commit after CI is updated